### DISCUSSION TITLE
IL2CPP line number does not show when using Unity Player (Android and IOS)

### DISCUSSION THREAD

**Discussion by frikic** [getsentry/sentry-unity#1524] [Q&A]
Hello,

We are developing Android and IOS game in Unity 2022.3.14f1 and we integrated sentry in our project,
following instructions provided by you.
Our idea was to get code lines with IL2CPP, which is enabled by default in your plugin, but we cant see
line numbers. Is there something additional that has to be done in order to make them work?

<img width="822" alt="image" src="https://github.com/getsentry/sentry-unity/assets/12951171/e6bfec46-20f3-4e14-9b4c-3f9741a8d3f2">

**Comment by bitsandfoxes** [comment]
Sorry for the late reply. This should definitely work.
It looks like the symbolication got stuck in the generated cpp. Could you share your `Awake` method?

**Reply by frikic** [reply]
Nothing fancy in Awake, just a null reference exception, intentionally made to test your system. 
I forgot exactly what I did I think I made fake Collider2D and then set enabled before actually assigning 
anything to it.
I can make a new one and send you exact code if you wish?

**Reply by frikic** [reply]
Hm I wrote also in [IOS & Android] Missing IL2CPP Line Number Information #1346, and got this answer from you there:

"A thousand apologies. I missed the notification on this one. I'll update our docs to properly reflect the current limitations.
But to summarize it here: Events going through the logger do not have line numbers. It's possible to have the stacktrace show up on the event by enabling AttachStacktrace on the options.
As of right now, the IL2CPP line number feature requires an exception to go through the exception processor and retrieve the instruction addresses from Unity's IL2CPP backend. This includes Debug.LogException since Unity omits the stacktrace on the exception. Instead, it's recommended to use SentrySdk.CaptureException instead."

So not sure is this is you answer, or it should work like you wrote here?
